.github/workflows/validation_trigger.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow triggers poliastro/validation CI from poliastro/poliastro
+# everytime a new commit is introduced in poliastro:master branch
+name: Validation test cases
+
+on:
+  # A collection of events which trigger the validation action
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  release:
+
+jobs:
+  
+  # Validate poliastro's new features
+  validation:
+    runs-on: ubuntu-latest
+
+    # Steps to be followed during validation job
+    steps:
+      - name: Trigger poliastro/validation
+        run: |
+          curl -X POST https://api.github.com/repos/poliastro/validation/dispatches \
+          -H 'Accept: application/vnd.github.everest-preview+json' \
+          -u ${{ secrets.VALIDATION_TRIGGER_TOKEN }} \
+          --data '{"event_type": "ping-validation", "client_payload": { "repository": "'"$GITHUB_REPOSITORY"'" }}'
```

Comment:
I'm not sure how to proceed. What value should `VALIDATION_TRIGGER_TOKEN` have? Is it something that https://github.com/poliastro/validation/ is expecting? If so, how did you configure it? Is the `ping-validation` something that you should have added in https://github.com/poliastro/validation/pull/11 ? I guess you're using this API https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#create-a-repository-dispatch-event, but I don't see the `-u` parameter.

Reply:
The `secrets.TOKEN_NAME` variables are used to reference stored secrets without explicitly calling its value. Thus, a `VALIDATION_TRIGGER_TOKEN` should be created. However, how I see your link to official docs, I am not sure if we require a `token` for this simple task of "pinging" the validation repository.

Regarding the "ping-validation" variable, it is defined within the JSON data one. When complex workflows are defined, those might be useful for debugging or reducing non-useful tasks. For example: if the validation repo actions is triggered by a change in poliastro repo, we now the `code_quality` job should not be applied (since no code has been modified within validation). Thus, we might include add the following to the validation actions file:

```YAML
if github.event.action == "ping-validation"
# do not run the code quality test
```

Finally, parameter -u is the one used for passing user credentials (name + password) within the `curl` command. 

Let me try to refactor this without tokens at all in a new commit and see what happens.

Reply:
This comment might be interesting: https://github.community/t/triggering-by-other-repository/16163/6. They claim that token minimum permission is `public_repo`

Reply:
Hmm so we indeed have to "fake" that the webhook is being called by a user (it can be either you and I) and then we have to create a personal access token as you suggested earlier... https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token okay, I didn't expect that but it makes sense I guess!

Reply:
Resolved!